src/components/Navbar/Menu.js

```diff
@@ -9,9 +9,7 @@ import styles from './Menu.module.scss'
 const Menu = ({ isAuthenticated, redirectTo }) => {
   const dispatch = useDispatch()
 
-  const handleSignOut = () => {
-    console.log('Signing Out')
-  }
+  const handleSignOut = () => {}
```

Comment:
Why is this function empty? Can we add the signout action and thispatch it from here?